scripts/set-default-contact/index.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
+const request = require('request-promise-native');
+const express = require('express');
+
+const app = express();
+app.use(express.json());
+
+const compileUrl = path => {
+  try {
+    return new URL(path, process.env.COUCH_URL);
+  } catch(e) {
+    throw new Error(`Error in compileUrl() while creating URL from ${process.env.COUCH_URL}${path},` +
+          ` error is: ${e.message}`);
+  }
+};
+
+const getChtUsers = async () => {
+  console.log('   Using URL taken from COUCH_URL env var: ', process.env.COUCH_URL);
+  console.log('');
+  const url = compileUrl('/api/v2/users');
+  const options = {
+    uri: url.href,
+    json: true
+  };
+
+  return request.get(options)
+    .then(users => {
+      if (typeof users === 'object' ){
+        console.log('   Found', users.length, 'users\n');
+        return users;
+      }
+    });
+};
+
+const getObjectFromMedicDb = async id => {
+  const url = compileUrl('/medic/' + id);
+  const options = {
+    uri: url.href,
+    json: true
+  };
+
+  return request.get(options)
+    .then(object => {
+      return object;
+    });
+
+};
+
+const hasDefaultContact = async user => {
+  if(typeof user.place === 'object' && typeof user.contact === 'object' &&  user.place._id  ){
+    const place = await getObjectFromMedicDb(user.place._id);
+    return !(typeof place.contact === 'object' && !place.contact._id);
+  } else {
+    // return true for invalid users like admin or medic so we don't process them
+    return true;
+  }
```

Comment:
No need for this `else` here
```suggestion
  if(typeof user.place === 'object' && typeof user.contact === 'object' &&  user.place._id  ){
    const place = await getObjectFromMedicDb(user.place._id);
    return !(typeof place.contact === 'object' && !place.contact._id);
  }
  // return true for invalid users like admin or medic so we don't process them
  return true;
```

Reply:
ah - yes - at one point my IDE refactored something similar for implicit `return true` I think or something?  Anyway - thanks for the tip!